weave

```diff
@@ -1552,6 +1571,9 @@ launch_proxy() {
         --entrypoint=/home/weave/weaveproxy \
         $WEAVEPROXY_DOCKER_ARGS $EXEC_IMAGE $COVERAGE_ARGS $PROXY_ARGS)
     wait_for_status $PROXY_CONTAINER_NAME http_call_unix $PROXY_CONTAINER_NAME status.sock
+    for sock in $PROXY_VAR_RUN_SYMLINKS; do
+      ln -sf -T "/var/run/weave/$sock" "/host/var/run/$sock"
```

Comment:
The intent is to link from `/var/run/weave` to `/var/run` is that correct? If so, the arguments to `ln` are reversed...